cpp/test/sg/rf_treelite_test.cu

```diff
@@ -274,23 +277,22 @@ class RfTreeliteTestClf : public RfTreeliteTestCommon<T, L> {
     // #class for multi-class classification
     this->task_category = 2;
 
-    float *weight, *noise, *temp_label_d;
+    float *weight, *temp_label_d;
```

Comment:
`weight` and `temp_label_d` must be freed with `cudaFree()` at the end of this function.